backend/tests/unit/dal/Entities/Extensions/ParcelExtensionsTest.cs

```diff
@@ -0,0 +1,412 @@
+using FluentAssertions;
+using Pims.Core.Extensions;
+using Pims.Dal.Entities;
+using Pims.Dal.Entities.Helpers.Extensions;
+using System;
+using System.Diagnostics.CodeAnalysis;
+using Xunit;
+
+namespace Pims.Dal.Test.Entities
+{
+    [Trait("category", "unit")]
+    [Trait("category", "dal")]
+    [Trait("category", "entities")]
+    [Trait("group", "entity")]
+    [ExcludeFromCodeCoverage]
+    public class ParcelExtensionsTest
+    {
+        #region Tests
+        #region GetMostRecentEvaluation
+        [Fact]
+        public void GetMostRecentEvaluation_Value()
+        {
+            // Arrange
+            var date = DateTime.UtcNow;
+            var building = new Parcel();
+            building.Evaluations.Add(new ParcelEvaluation(building, date, EvaluationKeys.Assessed, 111.11m));
+            building.Evaluations.Add(new ParcelEvaluation(building, date.Subtract(new TimeSpan(1, 0, 0)), EvaluationKeys.Assessed, 222.22m));
```

Comment:
date syntax with the timespan is a bit harder to read then the date.addYears used in the building extension tests.

Reply:
I agree, I'll update.